src/main/java/com/iexec/core/chain/adapter/BlockchainAdapterService.java

```diff
@@ -53,16 +53,13 @@ public BlockchainAdapterService(BlockchainAdapterClient blockchainAdapterClient)
      */
     public Optional<String> requestInitialize(String chainDealId, int taskIndex) {
         try {
-            ResponseEntity<String> initializeResponseEntity =
-                    blockchainAdapterClient.requestInitializeTask(chainDealId, taskIndex);
-            if (initializeResponseEntity.getStatusCode().is2xxSuccessful()
-                    && !StringUtils.isEmpty(initializeResponseEntity.getBody())) {
-                String chainTaskId = initializeResponseEntity.getBody();
+            String chainTaskId = blockchainAdapterClient.requestInitializeTask(chainDealId, taskIndex);
+            if (!StringUtils.isEmpty(chainTaskId)) {
                 log.info("Requested initialize [chainTaskId:{}, chainDealId:{}, " +
                         "taskIndex:{}]", chainTaskId, chainDealId, taskIndex);
                 return Optional.of(chainTaskId);
             }
-        } catch (Throwable e) {
+        } catch (Exception e) {
```

Comment:
`FeignException`? -> https://github.com/iExecBlockchainComputing/iexec-core/blob/325115f2281e587ab0564eced4ca4e36ae030a95/src/main/java/com/iexec/core/chain/adapter/BlockchainAdapterService.java#L165

Reply:
Are we sure that `FeignException` will be the only type of `RuntimeException` thrown ?
Wouldn't it be better to use `RutimeException` as an overall way to cacth most of happening exceptions during runtime ?

Reply:
1. I thought so, but did not find the information in the doc, we can use the generic one if you prefer yes